website/pages/docs/overlay/modal.mdx

```diff
@@ -299,7 +299,7 @@ function VerticallyCenter() {
 
 The `Modal` comes with a scale transition by default but you can change it by
 passing a `motionPreset` prop, and set its value to either `slideInBottom`,
-`slideInRight`, or `scale`.
+`slideInRight`, or `scale`, or `none`.
```

Comment:
```suggestion
`slideInRight`, `scale` or `none`.
```

Reply:
Lets omit the additional "or" ✏️

Reply:
I must have been drunk. Fix is now being pushed.